servicex_app/servicex/code_gen_adapter.py

```diff
@@ -48,16 +47,44 @@ def generate_code_for_selection(
         from io import BytesIO
         from zipfile import ZipFile
 
+        postObj = {}
+        if transformer_image:
+            postObj = {
+                "code": request_record.selection,
+                "transformer_image": transformer_image,
+            }
+        else:
+            postObj = {
+                "code": request_record.selection,
+            }
+
         result = requests.post(self.code_gen_url + "/servicex/generated-code",
-                               data=request_record.selection)
+                               json=postObj)
 
         if result.status_code != 200:
             msg = result.json()['Message']
             raise ValueError(f'Failed to generate translation code: {msg}')
 
-        zipfile = ZipFile(BytesIO(result.content))
+        print(result.data)
```

Comment:
Don't leave debugging statement in code

Reply:
removed the print statement, I will remove debug statements in further commits.